repository/user.go

```diff
@@ -0,0 +1,55 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/georgysavva/scany/pgxscan"
+	"github.com/serhiihuberniuk/blog-api/models"
+)
+
+func (r *Repository) CreateUser(ctx context.Context, user *models.User) error {
+	sql := "INSERT INTO users VALUES ($1, $2, $3, $4, $5)"
+
+	_, err := r.Db.Exec(ctx, sql, user.ID, user.Name, user.Email, user.CreatedAt, user.UpdatedAt)
+	if err != nil {
+		return fmt.Errorf("cannot create user: %w", err)
+	}
+
+	return nil
+}
+
+func (r *Repository) GetUser(ctx context.Context, userID string) (*models.User, error) {
+	var user models.User
+
+	sql := "SELECT * FROM users WHERE id=$1"
```

Comment:
please do not write `SELECT *`, you have to define all columns you want to select: `SELECT column1, column2...`